From 0b209f5946d423a0d58f1e24a53669abef6a5a3f Mon Sep 17 00:00:00 2001
From: Marcin Wojtas <mw@semihalf.com>
Date: Tue, 15 Oct 2019 05:42:29 +0200
Subject: [PATCH 0009/1239] arm64: mvebu: a8k: config: Add missing config
 options

This patch adds missing config options:
- For A8k DB config file, enable CONFIG_CMD_GPIO, CONFIG_NAND,
  CONFIG_CMD_REGULATOR, CONFIG_DM_GPIO, CONFIG_DM_REGULATOR_FIXED
  and CONFIG_DM_REGULATOR_GPIO, CONFIG_NAND_PXA3XX, CONFIG_DM_PCA953X;

Change-Id: Id33f268212df4478a99e11d222e4e433fcf87f0c
Signed-off-by: Omri Itach <omrii@marvell.com>
Signed-off-by: Marcin Wojtas <mw@semihalf.com>
---
 configs/mvebu_db_armada8k_defconfig | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/configs/mvebu_db_armada8k_defconfig b/configs/mvebu_db_armada8k_defconfig
index c4b1233de4..f07ec8aa3a 100644
--- a/configs/mvebu_db_armada8k_defconfig
+++ b/configs/mvebu_db_armada8k_defconfig
@@ -23,8 +23,10 @@ CONFIG_ARCH_EARLY_INIT_R=y
 CONFIG_BOARD_EARLY_INIT_F=y
 CONFIG_SYS_PROMPT="Marvell>> "
 # CONFIG_CMD_FLASH is not set
+CONFIG_CMD_GPIO=y
 CONFIG_CMD_I2C=y
 CONFIG_CMD_MMC=y
+# CONFIG_CMD_NAND is not set
 CONFIG_CMD_PCI=y
 CONFIG_CMD_SF=y
 CONFIG_CMD_SPI=y
@@ -34,17 +36,22 @@ CONFIG_CMD_TFTPPUT=y
 CONFIG_CMD_CACHE=y
 CONFIG_CMD_TIME=y
 CONFIG_CMD_MVEBU_BUBT=y
+CONFIG_CMD_REGULATOR=y
 CONFIG_CMD_EXT4_WRITE=y
 CONFIG_MAC_PARTITION=y
 CONFIG_DEFAULT_DEVICE_TREE="armada-8040-db"
 CONFIG_ENV_IS_IN_SPI_FLASH=y
 CONFIG_AHCI_MVEBU=y
+CONFIG_DM_GPIO=y
+CONFIG_DM_PCA953X=y
 CONFIG_DM_I2C=y
 CONFIG_SYS_I2C_MVTWSI=y
 CONFIG_MISC=y
 CONFIG_DM_MMC=y
 CONFIG_MMC_SDHCI=y
 CONFIG_MMC_SDHCI_XENON=y
+CONFIG_NAND=y
+CONFIG_NAND_PXA3XX=y
 CONFIG_SPI_FLASH=y
 CONFIG_SF_DEFAULT_MODE=0
 CONFIG_SPI_FLASH_MACRONIX=y
@@ -60,6 +67,8 @@ CONFIG_PCIE_DW_MVEBU=y
 CONFIG_MVEBU_COMPHY_SUPPORT=y
 CONFIG_PINCTRL=y
 CONFIG_PINCTRL_ARMADA_8K=y
+CONFIG_DM_REGULATOR_FIXED=y
+CONFIG_DM_REGULATOR_GPIO=y
 CONFIG_DEBUG_UART_SHIFT=2
 CONFIG_DEBUG_UART_ANNOUNCE=y
 CONFIG_SYS_NS16550=y
-- 
2.29.0

